experimental/devices/ina219/ina219.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package ina219
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/physic"
+)
+
+// Option is a configuration function
```

Comment:
Please use Opts struct pattern for coherency with the other drivers. This can be done in a follow up.